libr/util/print.c

```diff
@@ -514,12 +514,20 @@ R_API void r_print_byte(RPrint *p, const char *fmt, int idx, ut8 ch) {
 R_API int r_print_string(RPrint *p, ut64 seek, const ut8 *buf, int len, int options) {
 	int i;
 	bool wide = (options & R_PRINT_STRING_WIDE);
+	bool wide32 = (options & R_PRINT_STRING_32WIDE);
 	bool zeroend = (options & R_PRINT_STRING_ZEROEND);
 	bool wrap = (options & R_PRINT_STRING_WRAP);
 	bool urlencode = (options & R_PRINT_STRING_URLENCODE);
 	p->interrupt = 0;
 	int col = 0;
 	for (i = 0; !p->interrupt && i < len; i++) {
```

Comment:
please skip the null bytes if any at the begining of the string to allow to print unaligned wide strings